documentation/modules/rn-2.5.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-2.5_{context}"]
+= {project-full} 2.5
+
+You can use {the-lc} {project-first} to migrate virtual machines from the following source providers to {virt} destination providers:
+
+* VMware vSphere
+* {rhv-full} ({rhv-short})
+* {osp}
+* Open Virtual Appliances (OVAs) that were created by VMware vSphere
+* Remote {virt} clusters
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-25_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration using OpenStack
+
+In this version, migration using OpenStack source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Disabling FIPS
+
+EMS enforcement is disabled for migrations with VMware vSphere source providers in order to enable migrations from versions of vSphere that are supported by {project-full} but do not comply with the 2023 FIPS requirements.
+
+.Integration of the create/update provider dialog
+
+ The forms of create and update provider are replaced with native pages within the {ocp-name}(OCP) console.
```

Comment:
```suggestion
 The forms of create and update provider are replaced with native pages within the {ocp-name}(OCP) console.
```

the end user does not "see" the change to native pages, it's an implementation detail,
but they do see a design change
we can say 
the forms of create and update provider where re designed to align with the ocp console design and added settings of the new Openstack authentication,
make VDDK optional 
allow use of system CA  
add OVA settings

Reply:
@yaacov ok, maybe I mixed things here - was it the create-plan dialog that wasn't integrated well into the console and triggered a separate window?

yeah, if the create/update provider dialog was already integrated into the console and "just" reshaped to align with the OCP console standard then this need to be rephrased

about the list of addition at the end - it's a release note, we don't have to list all those things as they are mentioned in separate items

Reply:
in this release we only updated the providers dialog, it was using none ocp api and ui components, so updating to use ocp api and components changed the way it looks and feel to align with other parts of the console, and some data inconsistencies like values not getting updated was fixed

Reply:
ok, so let's try to summarize it in a release-note way:
```suggestion
Reworked the forms of create and update providers so that they will align with the typical look and feel within the OCP console and ensure the latest data is presented in these dialogs.
```

Reply:
or actually, we have [a bug on the second part](https://issues.redhat.com/browse/MTV-603) so we can go with
```suggestion
 Reworked the forms of create and update providers so that they will align with the typical look and feel within the OCP console.
```